azurelinuxagent/daemon/scvmm.py

```diff
@@ -70,4 +71,5 @@ def run(self):
         if self.detect_scvmm_env():
             self.start_scvmm_agent()
```

Comment:
Why are you calling self.start_scvmm_agent() a second time? If detect_scvmm_env returned true, then that function already called start_scvmm_agent (back on line 53).